### DISCUSSION TITLE
Translate part of display

### DISCUSSION THREAD

**Discussion by Perek101** [pedroSG94/RootEncoder#1387] [Q&A]
Hello, Pedro. I have 3 questions:
1) I need to stream an RTSP VLC stream (planning to use VLCVideoLayout to display the RTSP stream). Is there a way to do this?
2) Is it possible to use your implementation of display translation to only transmit a part of it? 
3) How can I start your RTSP server without passing video info (sps, pps, vps) to it? I want to start the server and wait for the RTSP stream from another device.

**Comment by pedroSG94** [comment]
Hello,

1 - As you suggested you can capture your device screen with DisplayRtmp, DisplayRtsp, etc or find a way to get raw h264 video from VLC and stream it directly to RtspClient
2 - Actually you can't do it. [You can develop a filter](https://github.com/pedroSG94/RootEncoder/wiki/Real-time-filters#create-your-own-filter) to do it but I haven't a filter already developed to do it
3 - You can't do it. The project is developed only to send video/audio to a RTSP player without use a media server. That project don't  support receive video/audio.

**Reply by Perek101** [reply]
I formulated the first question a little incorrectly. I need to run RTSP server with VLC stream
